.github/workflows/release-tag.yml

```
@@ -0,0 +1,37 @@
+jobs:
+    build:
+        name: Create Release
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@master
+
+            - name: Get the release version from the tag
+              shell: bash
+              if: env.RELEASE_VERSION == ''
+              run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:52: Double quote to prevent globbing and word splitting

(shellcheck)